docs/topics/method-design.adoc

```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
 // * docs/getting-started-guide/master.adoc
-[id='design_phase_{context}']
+[id='method-design_{context}']
```

Comment:
Global change: There should be a blank line between the comment line and the id line:

```
// Module included in the following assemblies:
// * docs/getting-started-guide/master.adoc

[id='method-design_{context}']
```

Also, there should be a `//` line between the `/ Module included...` line and the `// * docs/...` line:

```
// Module included in the following assemblies:
//
// * docs/getting-started-guide/master.adoc
```


You might be able to do it with a global search/replace if you are careful and can use regex.